deploy/wasmplugin.yaml

```diff
@@ -0,0 +1,12 @@
+ apiVersion: extensions.istio.io/v1alpha1
+ kind: WasmPlugin
+ metadata:
+   name: demo
+   namespace: istio-system
+ spec:
+    selector:
+      matchLabels:
+        app: toystore
+    url: <replace-with-wasm-oci-or-remote-url>
+    pluginConfig:
+      auth_cluster: "outbound|50051||authorino-authorization.default.svc.cluster.local"
```

Comment:
When Authorino is deployed by the operator it usually [names the service](https://github.com/Kuadrant/authorino-operator/blob/05cf2ad0c76b6138298f30fbca19da4a2c9b2cf1/controllers/authorino_controller.go#L681) `<name-of-authorino-cr>-authorino-authorization`. This means that the ext authz service would be reachable at `authorino-authorino-authorization.default.svc.cluster.local` for the `Authorino` CR included here.

Reply:
Thanks for the find! Fixed.